spec/lib/aws_config_spec.rb

```diff
@@ -2,7 +2,11 @@
 
 describe AWSConfig do
   let(:sample_config_file) { File.expand_path("../../samples/config.txt", __FILE__) }
-  before { AWSConfig.config_file = sample_config_file }
+  let(:sample_creds_file) { File.expand_path("../../samples/credentials.txt", __FILE__) }
```

Comment:
Sorry, I haven't configured Hound CI appropriately, and this rule seems outdated. So please ignore them.